index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sydney's Calculator</title>
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="stylesheet.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+</head>
+
+<body>
+    <div class="intro">
+        <h1>Web Calculator</h1>
+    </div>
+    <div class="inputBox">
+
+        <input type="number" id="firstNumber" />
+        <span>+</span>
+        <input type="number" id="secondNumber" />
+        <span>=</span>
+        <input type="button" value="Submit" style="margin: auto;" onclick="addNumbers()" />
```

Comment:
please refrain from styling in-line here, especially since you have a css file already.